src/imarker_end_effector.cpp

```diff
@@ -312,10 +312,7 @@ IMarkerEndEffector::makeBoxControl(visualization_msgs::InteractiveMarker& msg)
   marker.scale.x = msg.scale * 0.3;   // x direction
   marker.scale.y = msg.scale * 0.10;  // y direction
   marker.scale.z = msg.scale * 0.10;  // height
-  marker.color.r = 0.5;
-  marker.color.g = 0.5;
-  marker.color.b = 0.5;
```

Comment:
This isn't purple but grey...

Reply:
Yes, but I don't see an issue with this (tiny) change in behavior. If you disagree please change it before merging :cat2: